tests/test_tutorial.py

```diff
@@ -223,13 +223,10 @@ def test_tutorial(self):
 
 
     repository = load_repository('repository')
-    list_of_targets = repository.get_filepaths_in_directory(
-        os.path.join('repository', 'targets'), recursive_walk=False, followlinks=True)
 
-    self.assertListEqual(sorted(list_of_targets), [
-        os.path.abspath(os.path.join('repository', 'targets', 'file1.txt')),
-        os.path.abspath(os.path.join('repository', 'targets', 'file2.txt')),
-        os.path.abspath(os.path.join('repository', 'targets', 'file3.txt'))])
+    # List of targets is hardcoded since get_filepaths_in_directory()
```

Comment:
Is this a TODO item to switch to using `get_filepaths_in_directory` + a helper at some point? The comment doesn't make a lot of sense once the diff showing the deleted call to `get_filepaths_in_directory` isn't visible. What do you think?

Reply:
Agree, I tried to improve the comment and keep the TODO idem: https://github.com/theupdateframework/tuf/pull/1008/commits/6d8a84d9938d149e01c3bd0bfb52d056bb5b7b55